include/dca/parallel/mpi_concurrency/mpi_collective_sum.hpp

```diff
@@ -165,23 +201,23 @@ void MPICollectiveSum::sum(std::vector<scalar_type>& m) const {
   m = std::move(result);
 }
 
-template <typename scalar_type>
-void MPICollectiveSum::sum(std::map<std::string, std::vector<scalar_type>>& m) const {
-  typedef typename std::map<std::string, std::vector<scalar_type>>::iterator iterator_type;
-
-  iterator_type it = m.begin();
-
-  for (; it != m.end(); ++it) {
-    std::vector<scalar_type> values((it->second).size());
-
-    for (size_t l = 0; l < (it->second).size(); l++)
-      values[l] = (it->second)[l];
-
-    sum(values);
+// template <typename Scalar>
```

Comment:
Looks like it should just be deleted.

Reply:
removed in 7586540c6a2bd8e716e46595ff48c4b4f291fea3